src/styles/utils/resets/resets-default.scss

```diff
@@ -0,0 +1,17 @@
+
+@mixin reset-styles {
+  margin: 0;
+  padding: 0;
+}
+
+@mixin reset-styles-button {
+  border: none;
+  outline: none;
+}
+
+@mixin default-font-family {
+  font-family: $general-font-family;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
```

Comment:
why do you need mixins for reset, you will not reuse it, looks like overingenering